package.json

```diff
@@ -5,15 +5,18 @@
   "author": "Scratch Foundation",
   "version": "3.30.5",
   "license": "AGPL-3.0-only",
+  "main": "./dist/main/main.js",
   "scripts": {
     "clean": "rimraf ./dist ./static/assets",
-    "start": "mkdirp ./dist && electron-webpack dev --bail --display-error-details --env.minify=false --no-progress",
-    "compile": "mkdirp ./dist && electron-webpack --bail --display-error-details --env.minify=false --no-progress",
+    "start": "node scripts/start.js",
+    "compile:renderer": "webpack --config webpack.renderer.js",
+    "compile:main": "webpack --config webpack.main.js",
+    "compile": "npm run compile:renderer && npm run compile:main",
     "fetch": "rimraf ./static/assets/ && mkdirp ./static/assets/ && node ./scripts/fetchMediaLibraryAssets.js",
     "build": "npm run build:dev",
-    "build:dev": "npm run compile && npm run doBuild -- --mode=dev",
-    "build:dir": "npm run compile && npm run doBuild -- --mode=dir",
-    "build:dist": "npm run compile && npm run doBuild -- --mode=dist",
+    "build:dev": "NODE_ENV=production npm run compile && npm run doBuild -- --mode=dev",
+    "build:dir": "NODE_ENV=production npm run compile && npm run doBuild -- --mode=dir",
+    "build:dist": "NODE_ENV=production npm run compile && npm run doBuild -- --mode=dist",
```

Comment:
Why is `NODE_ENV=production` necessary here?

Assuming it is necessary, could you set this up to use `crossenv` instead? That will allow Windows users to run these commands, too.